src/unix/udp.c

```diff
@@ -1312,7 +1312,7 @@ static int uv__udp_sendmsgv(int fd,
   nsent = 0;
 
 #if defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__) || \
-  (defined(__sun__) && defined(MSG_WAITFORONE))
+  (defined(__sun__) || defined(__QNX__) && defined(MSG_WAITFORONE))
```

Comment:
Breaks `__sun__`.  `&&` binds tighter than `||` so this is interpreted as "sun OR (qnx and msg_waitforone)"